.gitignore

```diff
@@ -224,3 +224,7 @@ dmypy.json
 
 # profiling data
 .prof
+
+# include assets
+!assets/**/*.png
+!assets/**/*.svg
```

Comment:
These should be at the top under the Custom section.

Reply:
Yup sounds good. That's now done.